### ISSUE TITLE
Is it possible to create custom operators on Angular-QueryBuilder?

### ISSUE THREAD

**Issue by yaelilola** [zebzhao/Angular-QueryBuilder#245]
I saw it is possible on https://querybuilder.js.org/#operators, but I'm trying to use Angular-QueryBuilder. Thanks!

**Comment by dmshapoval** [comment]
yes, it is possible. This package simply builds rules/rulesets. It will use operators which you provided in config, but how to treat them you have to define in your code

**Comment by DevMetwaly** [comment]
#138
